extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php

```diff
@@ -11,7 +11,10 @@ class WikiaMapsMapController extends WikiaMapsBaseController {
 	 * @return Array
 	 */
 	public function getTileSets() {
-		$params = [];
+		$params = [
```

Comment:
It's hard to say no to tests :japanese_ogre: It's up to you to decide. You could add a test checking if there are `sort`, `limit` parameters in `$params` always and if there is the additional one `search` element when passed with the request.

But again: it's up to you. I might be guessing it's additional value for you to spend some time with PHPUnit and a test is always an additional value for our code quality.